src/EnergyPlus/HeatBalanceManager.cc

```diff
@@ -5173,6 +5175,89 @@ namespace HeatBalanceManager {
         GetSpaceData(state, ErrorsFound);
     }
 
+    void GetIncidentSolarMultiplier(EnergyPlusData &state, bool &ErrorsFound)
+    {
+        auto &cCurrentModuleObject = state.dataIPShortCut->cCurrentModuleObject;
+        cCurrentModuleObject = "SurfaceProperty:IncidentSolarMultiplier";
+
+        constexpr const char *RoutineName("GetIncidentSolarMultiplier: ");
+
+        state.dataSurface->TotSurfIncSolMultiplier = state.dataInputProcessing->inputProcessor->getNumObjectsFound(state, cCurrentModuleObject);
+        if (state.dataSurface->TotSurfIncSolMultiplier > 0) {
```

Comment:
One thing I like to do in these situations is `If (!X) return;` This un-indents the rest of the function by one tab and makes it easier to read IMO.